ScoutSuite/providers/gcp/configs/base.py

```diff
@@ -1,230 +1,10 @@
-# -*- coding: utf-8 -*-
-
-# Python2 vs Python3
-try:
-    from Queue import Queue
-except ImportError:
-    from queue import Queue
-
-import itertools
-import json
-import re
-
-from google.api_core.exceptions import PermissionDenied
-from google.cloud import container_v1
-from googleapiclient.errors import HttpError
-
-from ScoutSuite.core.console import print_exception, print_error
-
 from ScoutSuite.providers.base.configs.base import BaseConfig
-from ScoutSuite.providers.gcp.utils import gcp_connect_service
 
 class GCPBaseConfig(BaseConfig):
```

Comment:
Can't we just delete this file completely, since we aren't using configs anymore?

Reply:
True